discoverable/README.md

```diff
@@ -0,0 +1,30 @@
+## How to craft discoverable repos
+
+For your repositories to flourish, it's important that people know they exist! Here are some tips for crafting discoverable repositories. 
+
```

Comment:
Use a descriptive repository name
Give your repository a description
Provide a URL for the deployment of the app, or documentation about this project